### PR TITLE
insight chain network switcher

### PR DESCRIPTION
**Pull request by micahriggan**
* get the supported chain and networks from bitcore-node
 * /api/status/enabled-chains
*  switch chains and networks from insight

### CODE REVIEW

File: packages/bitcore-node/src/routes/index.ts [hunk 1]
```diff
@@ -44,11 +44,16 @@ function getRouterFromFile(path) {
   });
 
   var route = require('./' + path);
+  console.log('using', route.path);
```
Comment (micahriggan) [inline packages/bitcore-node/src/routes/index.ts]: Remove this line

File: packages/insight/app/src/components/head-nav/head-nav.ts [hunk 2]
```diff
@@ -69,7 +71,7 @@ export class HeadNavComponent {
                 console.log('addr', data);
                 let parsedData: any = JSON.parse(data._body);
                 this.navCtrl.push('address', {
-                  selectedCurrency: this.currency.selectedCurrency,
+                  chain: this.apiProvider.selectedChain, network: this.apiProvider.selectedNetwork,
```
Comment (micahriggan) [inline packages/insight/app/src/components/head-nav/head-nav.ts]: formatting

File: packages/insight/app/src/pages/block-detail/block-detail.ts [hunk 3]
```diff
@@ -11,54 +12,59 @@ import { BlocksProvider } from '../../providers/blocks/blocks';
...
-            (err) => {
-                console.log('err is', err);
-                this.loading = false;
-            }
-        );
-    }
+  public ionViewDidLoad(): void {
+    this.blockProvider.getBlock(this.blockHash).subscribe(
+      data => {
+        console.log(data.block);
```
Comment (SonicWizard) [inline packages/insight/app/src/pages/block-detail/block-detail.ts]: Remove console.log

File: packages/insight/app/src/pages/broadcast-tx/broadcast-tx.ts [hunk 4]
```diff
@@ -4,9 +4,10 @@ import { ToastController } from 'ionic-angular';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Http } from '@angular/http';
 import { ApiProvider } from '../../providers/api/api';
+import { NavParams } from 'ionic-angular/navigation/nav-params';
 
 @IonicPage({
-  segment: ':selectedCurrency/broadcast-tx'
+  segment: ':selectedCurrency/:selectedNetwork/broadcast-tx'
```
Comment (SonicWizard) [inline packages/insight/app/src/pages/broadcast-tx/broadcast-tx.ts]: Do you want to change this to `:chain/:network` to be consistent with other segment definitions?

**Pull request merged**